src/hooks/useGetOrders.tsx

```diff
@@ -117,6 +118,42 @@ export function useGetTxOrders(txHash: string): Result {
   return { orders, error, isLoading }
 }
 
+export function useTxOrderExplorerLink(txHash: string): ExplorerLinkProps | Record<string, never> {
+  const networkId = useNetworkId() || undefined
+  const [isGPv2Tx, setGPv2Tx] = useState(false)
+  const [explorerLink, setExplorerLink] = useState({})
+
+  const fetchTxOrders = useCallback(
+    async (_txHash: string, networkId: Network): Promise<void> => {
+      const { config: networkIdList } = CONFIG.exchangeContractConfig
+      const availableNetworks = new Set(networkIdList.map(({ networkId }) => networkId))
+
+      for (const network of availableNetworks) {
+        try {
+          const ordersFetched = await getTxOrders({ networkId: Number(network), txHash: _txHash })
```

Comment:
I believe we'll have to do this check anyway when searching for a given tx.
I'm afraid we'll do several calls to the backend without really needing it.

Do you see what I mean?

Reply:
@alfetopito We have this logic here  https://github.com/gnosis/gp-ui/pull/933/files#diff-92ef89a8db4e59ad32891b00698921ea9e7fac10d9a70c702f0ca0a65322f6c6R195

Once that PR is merged, I'll update this hook in order to get that information directly from the function.